vectorstores/alloydb/vectorstores_test.go

```diff
@@ -0,0 +1,203 @@
+package alloydb_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/tmc/langchaingo/embeddings"
+	"github.com/tmc/langchaingo/internal/alloydbutil"
+	"github.com/tmc/langchaingo/llms/ollama"
+	"github.com/tmc/langchaingo/schema"
+	"github.com/tmc/langchaingo/vectorstores/alloydb"
+	"os"
+	"testing"
+)
+
+func getEnvVariables(t *testing.T) (string, string, string, string, string, string, string) {
+	t.Helper()
+
+	username := os.Getenv("ALLOYDB_USERNAME")
+	if username == "" {
+		t.Skip("ALLOYDB_USERNAME environment variable not set")
```

Comment:
nit: probably safer to fail instead of skip

Reply:
agree, t.Fatal is my preferred approach however I this project all vectorstore skip because running the whole test suit, you would need to pass the all the connection vars for all the databases, we may maintain the skip because of this!